### DISCUSSION TITLE
A Shipping Integration for ERPNext

### DISCUSSION THREAD

**Discussion by Jengro777** [karrioapi/karrio#406] [General]
https://github.com/frappe/erpnext-shipping

**Comment by danh91** [comment]
Hi @Avey777,

We will eventually build some key integrations. But it is not our current focus. 
We will be more than happy to help if you can build the integration for ERPNext.

**Reply by kj2222** [reply]
@danh91 we would be interested in building this integration. What support can your team provide?
